lib/analyzer/image-inspector.ts

```diff
@@ -1,8 +1,16 @@
 import * as Debug from "debug";
+import * as fs from "fs";
+import * as fx from "mkdir-recursive";
```

Comment:
why is it called fx?

Reply:
I got `fx` from the example in lib [repo](https://github.com/hex7c0/mkdir-recursive). I can change it to `mkdir-recursive`.
However you mentioned that you're afraid because of the license error. I can change the lib for another, do you recommend some? This one I got from Magma recommendation when I was discussing the changes in the `snyk-docker-pull`.  I'm just using it because I can't use `fs.mkdirSync({recursive: true})`, this `recursive options is not available on Node 8.

Reply:
I changed the lib because of the license complaints. I won't be accept by the CLI.